lib/modules/login/components/form.dart

```diff
@@ -0,0 +1,89 @@
+part of '../login_module.dart';
+
+class _Form extends StatefulWidget {
+  @override
+  __FormState createState() => __FormState();
+}
+
+class __FormState extends State<_Form> {
+  _LoginViewImplState get _state => context.findAncestorStateOfType()!;
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.fromLTRB(24, 0, 24, 0),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          _emailTextField(),
+          const SizedBox(
+            height: 20,
+          ),
+          _passwordTextField(),
+          const SizedBox(
+            height: 20,
+          ),
+          StreamsSelector0<bool>.value(
+            stream: _state._isLoginButtonDisabled,
+            builder: (_, isValid, __) {
+              return Button(
+                key: LoginView.loginButtonKey,
+                title: AppLocalizations.of(context)!.loginScreenLoginButtonText,
+                isEnabled: isValid,
+                onPressed: () => _state.delegate?.loginButtonDidTap.add([
+                  _state._emailController.text,
+                  _state._passwordController.text
```

Comment:
I think we should rename `_isLoginButtonDisabled` ->`_isLoginButtonEnabled` to avoid confusing, att I think `isValid` is `isDisabled` instead of `isEnabled`